### ISSUE TITLE
Clustering

### ISSUE THREAD

**Issue by clincha** [clincha/clincha#81]
By upgrading my personal cluster

- I want to be able to reboot a node when I patch it with no downtime so that my applications can continue to run 24/7
- I want to share storage across multiple hosts so that my applications can continue to run 24/7
- I want to cluster my Proxmox nodes together so that I can learn about clustering and the benefits it brings
- I want to cluster my Proxmox nodes so that I can run a highly available Kubernetes cluster

**Comment by clincha** [comment]
This properly commenced last week. The server in Edinburgh was turned off and brought home. Right now, it's making the journey from our home to Bristol. There, it will join the other two servers and form the cluster.

From there I'll start testing out Ceph over gigabit connections.

**Comment by clincha** [comment]
Tomorrow the Edinburgh Proxmox host will be reimaged into the newest Bristol node. I will need a game plan for what I need to do once the node is ready. I can also probably do a few tasks beforehand to get myself prepped.

Before:
- [ ] Save Factorio saves and configuration
- [ ] Delete all pods in the cluster
- [ ] Destroy all the Kubernetes hosts
- [ ] Destroy all the storage
- [ ] Rewrite Terraform for new environment

After
- [ ] Join the node to the cluster
- [ ] Setup the data pools
- [ ] Setup the config pools
- [ ] Create Kubernetes hosts
- [ ] Create storage provisioner
- [ ] Test persistent volume
- [ ] Test network mount

**Comment by clincha** [comment]
I spent a couple of hours after work with dad putting in a graphics card and re-imaging Proxmox on the old Edinburgh server. This machine is now bri-s-03. Dad managed to put in a graphics card, install an OS, take the graphics card out, add in the HBA and then close it all up too. It was an enjoyable time I wish I had recorded but here are some photos instead :)

![IMG_2661-COLLAGE](https://user-images.githubusercontent.com/10735762/236001364-9b264015-dd49-48fa-8cc5-5864d7f385af.jpg)
